dist/fabric.js

```diff
@@ -6087,7 +6087,7 @@ fabric.Pattern = fabric.util.createClass(/** @lends fabric.Pattern.prototype */
      * @type Boolean
      * @default
      */
-    stateful: true,
+    stateful: false,
```

Comment:
default stateful to false. stateful was used in automatic way to detect a change in object modified at end of transformation. Now we detect it properly with the result of actions. Stateful is no more necessary if not for user needs that will activate it. ( detecting changes in fill, stroke and so on)